admin/client/src/components/ListGroup/LIstGroup.scss

```diff
@@ -0,0 +1,96 @@
+// List-group, based on Bootstraps list-group
+// TODO split out campaign specific css from generic list-group
+
+.list-group {
+  margin-left: -$spacer-y;
+  margin-right: -$spacer-y;
+  border-bottom: 1px solid $border-color-light;
+  margin-bottom: $spacer-y*.75;
+}
+
+.list-group-item {
+	padding-left: $spacer-y;
+	padding-right: $spacer-y;
+	min-height: 64px;
+	cursor: pointer;
+	text-decoration: none;
+
+	&:first-child {
+		border-top: none;
+	}
+	&:hover {
+		text-decoration: none;
+
+		.list-group-item--has-links {
+			display: block;
+			opacity: 1;
+		}
+	}
+
+	&.active {
+		background-color: $brand-primary;
+		color: #FFF;
+		opacity: 1;
+
+		.list-group-item--has-links,
+		.list-group-item__status {
+			color: #FFF;
+			opacity: 1;
+		}
+	}
+}
+
+.list-group-item__heading {
+	font-weight: 400;
+	font-size: 14px;
+	margin: 2px 0 5px;
+	color: $body-color;
+}
+
+.list-group-item__thumbnail {
+	width: 64px;
+	height: 64px;
+	display: block;
+	background: #ccc;
+	float: left;
+	margin: -12px 12px 0 -$spacer-y;
+}
+
+// Show linked items
+.list-group-item--has-links,
+.list-group-item--is-linked {
+	color: $brand-primary;
+	float: right;
+	font-size: $font-size-sm;
+	position: absolute;
+	right: $spacer-y;
+	top: 27px;
+	opacity: 0;
+	transition: opacity .2s ease-in-out;
+
+	.font-icon-link {
+		font-size: 16px;
+		position: relative;
+		top: 3px;
+		margin-right: 1px;
+	}
+}
+
+.list-group-item--published {
```

Comment:
Yup true but then it wouldn't be attached to the list-group-item? because published is a state which can also be applied outside of campaigns. I felt it was something I didn't completely know how to resolve in a nice way yet so made a comment at the top "// TODO split out campaign specific css from generic list-group"